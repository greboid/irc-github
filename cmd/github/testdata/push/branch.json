{
  "ref": "refs/heads/test_branch",
  "before": "0000000000000000000000000000000000000000",
  "after": "1593111bb58ad9fba738dc9862d5eaf0cb74fd0d",
  "repository": {
    "id": 296927666,
    "node_id": "MDEwOlJlcG9zaXRvcnkyOTY5Mjc2NjY=",
    "name": "irc-github",
    "full_name": "greboid/irc-github",
    "private": false,
    "owner": {
      "name": "greboid",
      "email": "greboid@users.noreply.github.com",
      "login": "greboid",
      "id": 189618,
      "node_id": "MDQ6VXNlcjE4OTYxOA==",
      "avatar_url": "https://avatars.githubusercontent.com/u/189618?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/greboid",
      "html_url": "https://github.com/greboid",
      "followers_url": "https://api.github.com/users/greboid/followers",
      "following_url": "https://api.github.com/users/greboid/following{/other_user}",
      "gists_url": "https://api.github.com/users/greboid/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/greboid/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/greboid/subscriptions",
      "organizations_url": "https://api.github.com/users/greboid/orgs",
      "repos_url": "https://api.github.com/users/greboid/repos",
      "events_url": "https://api.github.com/users/greboid/events{/privacy}",
      "received_events_url": "https://api.github.com/users/greboid/received_events",
      "type": "User",
      "site_admin": false
    },
    "html_url": "https://github.com/greboid/irc-github",
    "description": null,
    "fork": false,
    "url": "https://github.com/greboid/irc-github",
    "forks_url": "https://api.github.com/repos/greboid/irc-github/forks",
    "keys_url": "https://api.github.com/repos/greboid/irc-github/keys{/key_id}",
    "collaborators_url": "https://api.github.com/repos/greboid/irc-github/collaborators{/collaborator}",
    "teams_url": "https://api.github.com/repos/greboid/irc-github/teams",
    "hooks_url": "https://api.github.com/repos/greboid/irc-github/hooks",
    "issue_events_url": "https://api.github.com/repos/greboid/irc-github/issues/events{/number}",
    "events_url": "https://api.github.com/repos/greboid/irc-github/events",
    "assignees_url": "https://api.github.com/repos/greboid/irc-github/assignees{/user}",
    "branches_url": "https://api.github.com/repos/greboid/irc-github/branches{/branch}",
    "tags_url": "https://api.github.com/repos/greboid/irc-github/tags",
    "blobs_url": "https://api.github.com/repos/greboid/irc-github/git/blobs{/sha}",
    "git_tags_url": "https://api.github.com/repos/greboid/irc-github/git/tags{/sha}",
    "git_refs_url": "https://api.github.com/repos/greboid/irc-github/git/refs{/sha}",
    "trees_url": "https://api.github.com/repos/greboid/irc-github/git/trees{/sha}",
    "statuses_url": "https://api.github.com/repos/greboid/irc-github/statuses/{sha}",
    "languages_url": "https://api.github.com/repos/greboid/irc-github/languages",
    "stargazers_url": "https://api.github.com/repos/greboid/irc-github/stargazers",
    "contributors_url": "https://api.github.com/repos/greboid/irc-github/contributors",
    "subscribers_url": "https://api.github.com/repos/greboid/irc-github/subscribers",
    "subscription_url": "https://api.github.com/repos/greboid/irc-github/subscription",
    "commits_url": "https://api.github.com/repos/greboid/irc-github/commits{/sha}",
    "git_commits_url": "https://api.github.com/repos/greboid/irc-github/git/commits{/sha}",
    "comments_url": "https://api.github.com/repos/greboid/irc-github/comments{/number}",
    "issue_comment_url": "https://api.github.com/repos/greboid/irc-github/issues/comments{/number}",
    "contents_url": "https://api.github.com/repos/greboid/irc-github/contents/{+path}",
    "compare_url": "https://api.github.com/repos/greboid/irc-github/compare/{base}...{head}",
    "merges_url": "https://api.github.com/repos/greboid/irc-github/merges",
    "archive_url": "https://api.github.com/repos/greboid/irc-github/{archive_format}{/ref}",
    "downloads_url": "https://api.github.com/repos/greboid/irc-github/downloads",
    "issues_url": "https://api.github.com/repos/greboid/irc-github/issues{/number}",
    "pulls_url": "https://api.github.com/repos/greboid/irc-github/pulls{/number}",
    "milestones_url": "https://api.github.com/repos/greboid/irc-github/milestones{/number}",
    "notifications_url": "https://api.github.com/repos/greboid/irc-github/notifications{?since,all,participating}",
    "labels_url": "https://api.github.com/repos/greboid/irc-github/labels{/name}",
    "releases_url": "https://api.github.com/repos/greboid/irc-github/releases{/id}",
    "deployments_url": "https://api.github.com/repos/greboid/irc-github/deployments",
    "created_at": 1600539828,
    "updated_at": "2021-04-18T16:24:10Z",
    "pushed_at": 1625857624,
    "git_url": "git://github.com/greboid/irc-github.git",
    "ssh_url": "git@github.com:greboid/irc-github.git",
    "clone_url": "https://github.com/greboid/irc-github.git",
    "svn_url": "https://github.com/greboid/irc-github",
    "homepage": null,
    "size": 93,
    "stargazers_count": 0,
    "watchers_count": 0,
    "language": "Go",
    "has_issues": true,
    "has_projects": true,
    "has_downloads": true,
    "has_wiki": true,
    "has_pages": false,
    "forks_count": 1,
    "mirror_url": null,
    "archived": false,
    "disabled": false,
    "open_issues_count": 0,
    "license": {
      "key": "mit",
      "name": "MIT License",
      "spdx_id": "MIT",
      "url": "https://api.github.com/licenses/mit",
      "node_id": "MDc6TGljZW5zZTEz"
    },
    "forks": 1,
    "open_issues": 0,
    "watchers": 0,
    "default_branch": "master",
    "stargazers": 0,
    "master_branch": "master"
  },
  "pusher": {
    "name": "greboid",
    "email": "greboid@users.noreply.github.com"
  },
  "sender": {
    "login": "greboid",
    "id": 189618,
    "node_id": "MDQ6VXNlcjE4OTYxOA==",
    "avatar_url": "https://avatars.githubusercontent.com/u/189618?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/greboid",
    "html_url": "https://github.com/greboid",
    "followers_url": "https://api.github.com/users/greboid/followers",
    "following_url": "https://api.github.com/users/greboid/following{/other_user}",
    "gists_url": "https://api.github.com/users/greboid/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/greboid/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/greboid/subscriptions",
    "organizations_url": "https://api.github.com/users/greboid/orgs",
    "repos_url": "https://api.github.com/users/greboid/repos",
    "events_url": "https://api.github.com/users/greboid/events{/privacy}",
    "received_events_url": "https://api.github.com/users/greboid/received_events",
    "type": "User",
    "site_admin": false
  },
  "created": true,
  "deleted": false,
  "forced": false,
  "base_ref": "refs/heads/master",
  "compare": "https://github.com/greboid/irc-github/compare/test_branch",
  "commits": [

  ],
  "head_commit": {
    "id": "1593111bb58ad9fba738dc9862d5eaf0cb74fd0d",
    "tree_id": "3b8989df11931eef1a8c4ab5572f2689a76097dc",
    "distinct": true,
    "message": "Update README.md",
    "timestamp": "2021-04-18T17:24:08+01:00",
    "url": "https://github.com/greboid/irc-github/commit/1593111bb58ad9fba738dc9862d5eaf0cb74fd0d",
    "author": {
      "name": "Greg Holmes",
      "email": "greboid@users.noreply.github.com",
      "username": "greboid"
    },
    "committer": {
      "name": "GitHub",
      "email": "noreply@github.com",
      "username": "web-flow"
    },
    "added": [

    ],
    "removed": [

    ],
    "modified": [
      "README.md"
    ]
  }
}