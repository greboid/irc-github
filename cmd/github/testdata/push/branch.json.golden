[greboid/irc-github] greboid created branch test_branch from branch master - https://github.com/greboid/irc-github/compare/test_branch